components/style/values/specified/image.rs

```diff
@@ -372,21 +394,22 @@ impl ToCss for AngleOrCorner {
     }
 }
 
-impl Parse for AngleOrCorner {
-    fn parse(context: &ParserContext, input: &mut Parser) -> Result<Self, ()> {
-        if input.try(|input| input.expect_ident_matching("to")).is_ok() {
+impl AngleOrCorner {
+    fn parse(context: &ParserContext, input: &mut Parser, mode: CompatMode) -> Result<Self, ()> {
+        if let Ok(angle) = input.try(|i| Angle::parse(context, i)) {
```

Comment:
Sounds fine, feel free to do it as a followup or file an issue (though given it's just `s/Angle::parse/Angle::parse_with_unitless` thought it may be worth to just do it here).